pyproject.toml

```diff
@@ -94,23 +94,16 @@ filterwarnings = [
 ]
 
 [tool.ruff]
-select = [
-    "E",
-    "F", # flake8
+src = ["src"]
+lint.extend-select = [
     "D", # pydocstyle
 ]
-extend-ignore = [
+lint.ignore = [
     "D212", # multi-line-summary-first-line
 ]
-src = ["src"]
-unfixable = [
```

Comment:
This is now an "unsafe" fix, and has to be enabled with `--unsafe-fixes`. So it's no longer needed to put in in unfixable.